include/swift/SIL/SILDeclRef.h

```diff
@@ -23,6 +23,7 @@
 #include "swift/AST/ClangNode.h"
 #include "swift/AST/GenericSignature.h"
 #include "swift/AST/TypeAlignments.h"
+#include "swift/AST/ASTContext.h"
```

Comment:
This is needed to compile the arg `witness.getASTContext().SILOpts.EnableSerializePackage` in SILLinkage.h.